pelita/ui/tk_canvas.py

```diff
@@ -519,7 +522,9 @@ def draw_status_info(self, game_state):
         turn = firstNN(game_state.get("turn"), "–")
         layout_name = firstNN(game_state.get("layout_name"), "–")
 
-        roundturn = "Bot %s, Round % 3s/%s" % (turn, round, max_rounds)
+        bot_repr = {'–':'–', 0: 'Blue Team, Bot 0', 1: 'Red Team, Bot 0',
```

Comment:
Come on, we can't change the convention blue/red right now. That's deep inside the code everywhere. We thought about just coloring the name "Bot 0" or "Bot 1", but then decided against it. We did not try it, though, so it may be a good solution indeed. I tried to quickly discover how to color the text in TK and failed. Do you want to have a go at it?

Reply:
(Of course, I don’t want to change the convention now. It really is too late. But it is true that I have a hard time remembering which colour comes first. For the students it may be easier.)
Could be that it is not as annoying in a real-life game when it is much slower.
I think it is something to solve with a better UI.

Maybe like a traffic light:

![ezgif-1-576357de9b7c](https://user-images.githubusercontent.com/216179/61712965-c592c100-ad57-11e9-937b-db6b9b3ed2b3.gif)

Reply:
another idea I had about this is that we may show the bot only when the grid is on. You couldn't read it anyway at 15fps anyway even before when it was Bot 0/1/2/3, so maybe it could be only switched on for the debugging phase, when people are more likely to look at it?
To indicate who moves, couldn't we highlight with a yellow background the bot who is moving next (not the one who just moved, though)? What do you think?

Reply:
Yeah, the traffic light is not a bad idea at all. Doable in TK? With no flickering?

Reply:
> Yeah, the traffic light is not a bad idea at all. Doable in TK? With no flickering?

We’ll have to try …

Reply:
Closing this conversation and adding a new issue.